pkg/admission/machines.go

```diff
@@ -63,7 +63,7 @@ func (ad *admissionData) mutateMachines(ar admissionv1beta1.AdmissionReview) (*a
 		// * oldMachine has Initializers on it
```

Comment:
This comment isn't true anymore.

Also, can you describe why you removed the finalizer handling here?

Reply:
@xrstf not finalizer but initializer. Initializers has been removed from upstream k8s.io/api.

Reply:
Oh mea culpa. Still, the comment is now wrong, no?

Reply:
Yeah that's actually my nad. the commit message is wrong :/